miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/ContainerRestController.java

```diff
@@ -147,4 +153,53 @@ public void bulkDelete(@RequestBody(required = true) List<Long> ids) throws IOEx
     return MisoWebUtils.generateSpreadsheet(input, PartitionSpreadsheets::valueOf, request, response);
   }
 
+  @PostMapping(value = "/validate-serial-number")
+  public ResponseEntity<?> validateSerialNumber(@RequestBody String params, HttpServletRequest request, HttpServletResponse response,
```

Comment:
These `RestController` methods should all return DTOs, or annotate with `@HttpStatus(Status.NO_CONTENT)` to indicate that no data is returned on success.  `RestException` should be thrown for any errors